Plugins/BTCPayServer.RockstarDev.Plugins.TransactionCounter/Views/PublicCounter/Counter.cshtml

```diff
@@ -0,0 +1,131 @@
+@model BTCPayServer.RockstarDev.Plugins.TransactionCounter.ViewModels.CounterViewModel
```

Comment:
This page will be severely simplified when that `HtmlTemplate` refactoring is implemented.